### DISCUSSION TITLE
Time-traveling debugger with copilot?

### DISCUSSION THREAD

**Discussion by turian** [community/community#50154] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Could you please make a time-traveling debugger with copilot?

I'd like it to watch the entire execution state of a running program, and then allow me to retroactively debug it using natural language.

**Comment by w3beginners** [comment]
Could you please make a time-traveling debugger with copilot?

I'd like it to watch the entire execution state of a running program, and then allow me to retroactively debug it using natural language. get use of travel 

[https://travel.w3beginners.com/](https://travel.w3beginners.com/)
